tox.ini

```diff
@@ -6,18 +6,13 @@
 [tox]
 isolated_build = True
 envlist =
-    # Django 3.2.9 adds support for Python 3.10
-    py{37,38,39,310}-django{32}-dotp{11}-allauth53
-
-    # Django 4.0+ requires Python 3.8 or higher
-    py{38,39,310,311}-django{40,41,42}-dotp{11}-allauth53
+    py{38,39,310,311}-django{42}
```

Comment:
FWIW Django 4.1 is still supported according to https://www.djangoproject.com/download/

Reply:
Mainstream support for 4.1 ended on the 5th of April 2023 and 4.1 doesn't have a LTS version.
I think that is what @akx meant.

Reply:
Yes but it is still supported until Dec 2023.

I don't have any strong opinion but it is likely not onerous to support.

Reply:
I guess it comes down to whether we want to support non-LTS versions or not. 
I also think it shouldn't be much more effort to keep supporting 4.1.

Reply:
Yeah, I know 4.1 is still supported for 5 months, but I'd rather poke people towards upgrading to the LTS version sooner rather than later :)

Reply:
Hmm yeah - I'm a bit conflicted.

On one hand I can see the benefit of having a "we support the latest LTS and whatever version has mainstream support". This would force users of django-allauth-2fa to upgrade Django if they want the newest release.

On the other hand I'm not sure if it is our place as a third party application to poke people. As long as it doesn't inflict serious overhead in maintaining the package, giving users of the package time to get their ducks in a row before upgrading, seems like the more flexible way to go.

But when that is said, I'm inclined to give it a go and see if it actually does users of django-allauth-2fa problems.

Reply:
Yeah I was going to say that it isn't really the place of third party packages to do that *for supported versions*. But I guess people can just stay on old versions of django-allauth-2fa if they need it.

Reply:
Points taken – let's go with supporting 4.1 for now, it's no big deal 😄